mteb/tasks/PairClassification/multilingual/PawsXPairClassification.py

```diff
@@ -8,13 +8,13 @@
 
 class PawsX(MultilingualTask, AbsTaskPairClassification):
     metadata = TaskMetadata(
-        name="PawsX",
+        name="PawsXPairClassification", 
         dataset={
             "path": "google-research-datasets/paws-x",
             "revision": "8a04d940a42cd40658986fdd8e3da561533a3646",
             "trust_remote_code": True,
         },
-        description="",
+        description="{PAWS-X: A Cross-lingual Adversarial Dataset for Paraphrase Identification",
         reference="https://arxiv.org/abs/1908.11828",
         category="s2s",
         type="PairClassification",
```

Comment:
Do you not want to also rename the class? 
Also if we rename this, we should probably rename all corresponding result files. Unfortunately we cannot rename the models meta of other people so we'll still need to allow for that in the leaderboard code.